src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1121,44 +1092,24 @@ namespace HeatBalFiniteDiffManager {
         //                      May 2011  B. Griffith add logging and errors when inner GS loop does not converge
         //                      November 2011 P. Tabares fixed problems with adiabatic walls/massless walls and PCM stability problems
 
-        //       RE-ENGINEERED  na
-
         // PURPOSE OF THIS SUBROUTINE:
         // this routine controls the calculation of the fluxes and temperatures using
         //      finite difference procedures for
         //      all building surface constructs.
 
-        Real64 MaxDelTemp(0.0);
-
-        auto &ConstructFD = state.dataHeatBalFiniteDiffMgr->ConstructFD;
-        auto &SurfaceFD = state.dataHeatBalFiniteDiffMgr->SurfaceFD;
```

Comment:
[src/EnergyPlus/HeatBalFiniteDiffManager.cc:1133]:(style),[constVariable],Variable 'ConstructFD' can be declared as reference to const
[src/EnergyPlus/HeatBalFiniteDiffManager.cc:1134]:(style),[constVariable],Variable 'SurfaceFD' can be declared as reference to const